### PR TITLE
Add Puter

### PR DESCRIPTION
**Pull request by jelveh**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Review by kokomo123 — APPROVED** [review]
LGTM, No issues here pending other thoughts from contributors. Thanks for your contribution!

File: software/puter.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+description: A privacy-first personal cloud to keep all your files, apps, and games in one secure place, accessible from anywhere at any time.
+licenses:
+  - AGPL-3.0
+platforms:
+  - Nodejs
+  - Docker
+tags:
+  - File Transfer & Synchronization
+  - Remote Access
+  - File Transfer - Distributed Filesystems
```
Comment (kokomo123) [inline software/puter.yml:12]:
```suggestion
```
This tag refers to another list, think the other two will be fine 👍🏻

File: software/puter.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: Puter
+website_url: https://puter.com/
+description: A privacy-first personal cloud to keep all your files, apps, and games in one secure place, accessible from anywhere at any time.
```
Comment (nodiscc) [inline software/puter.yml:3]:
```suggestion
description: Web-based operating system designed to be feature-rich, exceptionally fast, and highly extensible
```

Better description

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by Rabenherz112 — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

File: software/puter.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: Puter
+website_url: https://puter.com/
+description: Web-based operating system designed to be feature-rich, exceptionally fast, and highly extensible
```
Comment (nodiscc) [inline software/puter.yml:3]:
```suggestion
description: Web-based operating system designed to be feature-rich, exceptionally fast, and highly extensible.
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**